packages/fuels-core/src/json_abi.rs

```diff
@@ -522,7 +525,9 @@ pub fn parse_param(param: &Property) -> Result<ParamType, Error> {
         // Simple case (primitive types, no arrays, including string)
         Ok(param_type) => Ok(param_type),
         Err(_) => {
-            if param.type_field.contains("struct") || param.type_field.contains("enum") {
+            if param.type_field.starts_with(STRUCT_KEYWORD)
```

Comment:
use `is_custom_type`

Reply:
The `is_custom_type` function should be move alongside the `Property` type definition, and some util methods on `Property` could be added to make the code more readable.